packages/svelte/src/internal/client/constants.js

```diff
@@ -41,3 +41,8 @@ export const ELEMENT_NODE = 1;
 export const TEXT_NODE = 3;
 export const COMMENT_NODE = 8;
 export const DOCUMENT_FRAGMENT_NODE = 11;
+
+export const VALID_EFFECT_PARENT = 0;
+export const EFFECT_ORPHAN = 1;
+export const UNOWNED_DERIVED_PARENT = 2;
+export const EFFECT_TEARDOWN = 3;
```

Comment:
Does it make sense to declare them here if they are and can be used only in one file?